hcx-pipeline-jobs/core/src/main/scala/org/swasth/dp/core/job/BaseJobConfig.scala

```diff
@@ -51,11 +51,11 @@ class BaseJobConfig(val config: Config, val jobName: String) extends Serializabl
   // Default output configurations
   val enrichedOutputTag: OutputTag[util.Map[String, AnyRef]] = OutputTag[util.Map[String, AnyRef]]("enriched-events")
 
-  val retryOutputTag: OutputTag[String] = OutputTag[String]("retry-events")
-  val retryTopic = config.getString("kafka.retry.topic")
+  val retryOutputTag: OutputTag[util.Map[String, AnyRef]] = OutputTag[util.Map[String, AnyRef]]("retry-events")
```

Comment:
Why did you change it to a Map?

Reply:
This is incorrectly updated, it should be String only. I corrected it.